app/assets/javascripts/asset-strings.js.erb

```diff
@@ -0,0 +1,12 @@
+window.LoginGov = window.LoginGov || {}
+<% image_keys = [
+    'idv/phone.png'
+] %>
+
+window.LoginGov.AssetStrings = {
+  images: {}
+};
+
+<% image_keys.each do |key| %>
+  window.LoginGov.AssetStrings.images['<%=key %>'] = '<%= ActionController::Base.helpers.image_path(key) %>';
+<% end %>
```

Comment:
Based on our previous conversation, I think we should be able to make this less opinionated about the particular kinds of assets, instead creating a single object which can contain any of the paths we might want in the client:

```suggestion
window.LoginGov = window.LoginGov || {};
window.LoginGov.AssetStrings = {};

<% keys = [
  'idv/phone.png'
] %>

<% keys.each do |key| %>
  window.LoginGov.AssetStrings['<%= ActionController::Base.helpers.j key %>'] = '<%= ActionController::Base.helpers.j ActionController::Base.helpers.asset_path key %>';
<% end %>
```

In my testing, this creates an object mapping the original path to the one including the digest:

![image](https://user-images.githubusercontent.com/1779930/87589366-1f03a680-c6b3-11ea-9bec-2b86c62af497.png)